src/module/config.js

```diff
@@ -1047,43 +1047,183 @@ SFRPG.CR_EXP_LEVELS = [
     153600, 204800, 307200, 409600, 614400, 819200, 1228800, 1638400
 ];
 
-SFRPG.statusEffectIcons = [
-    "systems/sfrpg/icons/conditions/asleep.png",
-    "systems/sfrpg/icons/conditions/bleeding.png",
-    "systems/sfrpg/icons/conditions/blinded.png",
-    "systems/sfrpg/icons/conditions/broken.png",
-    "systems/sfrpg/icons/conditions/burning.png",
-    "systems/sfrpg/icons/conditions/confused.png",
-    "systems/sfrpg/icons/conditions/cowering.png",
-    "systems/sfrpg/icons/conditions/dazed.png",
-    "systems/sfrpg/icons/conditions/dazzled.png",
-    "systems/sfrpg/icons/conditions/dead.png",
-    "systems/sfrpg/icons/conditions/deafened.png",
-    "systems/sfrpg/icons/conditions/dying.png",
-    "systems/sfrpg/icons/conditions/encumbered.png",
-    "systems/sfrpg/icons/conditions/entangled.png",
-    "systems/sfrpg/icons/conditions/exhausted.png",
-    "systems/sfrpg/icons/conditions/fascinated.png",
-    "systems/sfrpg/icons/conditions/fatigued.png",
-    "systems/sfrpg/icons/conditions/flatfooted.png",
-    "systems/sfrpg/icons/conditions/frightened.png",
-    "systems/sfrpg/icons/conditions/grappled.png",
-    "systems/sfrpg/icons/conditions/helpless.png",
-    "systems/sfrpg/icons/conditions/nauseated.png",
-    "systems/sfrpg/icons/conditions/offkilter.png",
-    "systems/sfrpg/icons/conditions/offtarget.png",
-    "systems/sfrpg/icons/conditions/overburdened.png",
-    "systems/sfrpg/icons/conditions/panicked.png",
-    "systems/sfrpg/icons/conditions/paralyzed.png",
-    "systems/sfrpg/icons/conditions/pinned.png",
-    "systems/sfrpg/icons/conditions/prone.png",
-    "systems/sfrpg/icons/conditions/shaken.png",
-    "systems/sfrpg/icons/conditions/sickened.png",
-    "systems/sfrpg/icons/conditions/staggered.png",
-    "systems/sfrpg/icons/conditions/stable.png",
-    "systems/sfrpg/icons/conditions/stunned.png",
-    "systems/sfrpg/icons/conditions/unconscious.png"
-];
+SFRPG.statusEffects = [
```

Comment:
With your changes to config.js, it might actually be worthwhile refactoring statusEffectIconMapping away entirely, and replace the checks inside the hasCondition, getCondition, and setCondition functions with a test against statusEffects.